pkg/nsx/services/realizestate/realize_state.go

```diff
@@ -54,12 +55,14 @@ func (service *RealizeStateService) CheckRealizeState(backoff wait.Backoff, inte
 		if err != nil {
 			return err
 		}
+		entitiesRealized := 0
 		for _, result := range results.Results {
 			if entityType != "" && result.EntityType != nil && *result.EntityType != entityType {
 				continue
 			}
```

Comment:
Would you think we can remove this part or even the entityType parameter? As if the entityType is passed, this part will skip the count of realized resource which makes the final entitiesRealized!=len(results.Results). And if there is no use case for entityType, maybe we can just remove this parameter in this function?

Reply:
Done